jest.config.utils.js

```diff
@@ -0,0 +1,186 @@
+    // maxWorkers: "50%",
+
+    // An array of directory names to be searched recursively up from the requiring module's location
+    moduleDirectories: ['node_modules'],
+
+    // An array of file extensions your modules use
+    moduleFileExtensions: ['js', 'json', 'ts', 'node'],
+
+    // A map from regular expressions to module names that allow to stub out resources with a single module
+    // moduleNameMapper: {},
+
+    // An array of regexp pattern strings, matched against all module paths before considered 'visible' to the module loader
+    // modulePathIgnorePatterns: [],
+
+    // Activates notifications for test results
+    notify: true,
+
+    // An enum that specifies notification mode. Requires { notify: true }
+    notifyMode: 'failure-change',
+
+    // A preset that is used as a base for Jest's configuration
+    preset: 'ts-jest',
+
+    // Run tests from one or more projects
+    // projects: null,
+
+    // Use this configuration option to add custom reporters to Jest
+    // reporters: undefined,
+
+    // Automatically reset mock state between every test
+    // resetMocks: false,
+
+    // Reset the module registry before running each individual test
+    // resetModules: false,
+
+    // A path to a custom resolver
+    // resolver: null,
+
+    // Automatically restore mock state between every test
+    // restoreMocks: false,
+
+    // The root directory that Jest should scan for tests and modules within
+    // rootDir: null,
+
+    // A list of paths to directories that Jest should use to search for files in
+    // roots: [
+    //   "<rootDir>"
+    // ],
+
+    // Allows you to use a custom runner instead of Jest's default test runner
+    // runner: "jest-runner",
+
+    // The paths to modules that run some code to configure or set up the testing environment before each test
+    // setupFiles: [],
+
+    // A list of paths to modules that run some code to configure or set up the testing framework before each test
+    // setupFilesAfterEnv: [],
+
+    // A list of paths to snapshot serializer modules Jest should use for snapshot testing
+    // snapshotSerializers: [],
+
+    // The test environment that will be used for testing
+    testEnvironment: 'node',
+
+    // Options that will be passed to the testEnvironment
+    // testEnvironmentOptions: {},
+
+    // Adds a location field to test results
+    // testLocationInResults: false,
+
+    // The glob patterns Jest uses to detect test files
+    testMatch: ['**/__tests__/**/*.(test).[t]s?(x)', '**/deleteUsers/**/*.(test).[t]s?(x)'],
+
+    // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
+    testPathIgnorePatterns: ['/node_modules/', 'user_transformation.integration.test.js'],
```

Comment:
We can run all the TS test scripts with this (including integration).
```suggestion
    testMatch: ['test/**/*.test.ts?(x)'],
    
    // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
    testPathIgnorePatterns: ['/node_modules/'],
```

Reply:
updated